python/ray/data/_internal/execution/streaming_executor_state.py

```diff
@@ -204,9 +205,12 @@ def initialize_progress_bars(self, index: int, verbose_progress: bool) -> int:
         Return the number of enabled progress bars created for this operator.
         """
         is_all_to_all = isinstance(self.op, AllToAllOperator)
+        is_ray_job = os.environ.get("RAY_JOB_ID") is not None
         # Only show 1:1 ops when in verbose progress mode.
-        progress_bar_enabled = DataContext.get_current().enable_progress_bars and (
-            is_all_to_all or verbose_progress
+        progress_bar_enabled = (
+            DataContext.get_current().enable_progress_bars
+            and (is_all_to_all or verbose_progress)
+            and not is_ray_job
```

Comment:
can we instead make the default  `enable_progress_bars` False for ray jobs? 
So users can still set it to True in case needed.